src/DependencyInjection/Terminal42UrlRewriteExtension.php

```diff
@@ -14,18 +14,26 @@
 
 use Symfony\Component\Config\FileLocator;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
-use Symfony\Component\DependencyInjection\Extension\Extension;
 use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
+use Symfony\Component\HttpKernel\DependencyInjection\ConfigurableExtension;
 
-class Terminal42UrlRewriteExtension extends Extension
+class Terminal42UrlRewriteExtension extends ConfigurableExtension
 {
     /**
      * {@inheritdoc}
      */
-    public function load(array $configs, ContainerBuilder $container): void
+    protected function loadInternal(array $mergedConfig, ContainerBuilder $container)
```

Comment:
What's the difference exactly here, maybe you can explain me sometime :)

Reply:
Well you have to use the `ConfigurableExtension` class if you have the bundle configuration and this method is just required by that abstract class.